src/main/java/Duke.java

```diff
@@ -1,10 +1,196 @@
+import command.Command;
+import command.CommandExecute;
+import exception.DukeException;
+import exception.ExceptionType;
+import file.FileManager;
+import tasks.Task;
+import tasks.TaskManager;
+
+import messages.Messages;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Scanner;
+
 public class Duke {
+
+
+    public static final String TASK_AT = "/at";
+    public static final String TASK_BY = "/by";
+
+    /*path for home directory*/
+    private static final String root = System.getProperty("user.dir");
+    private static final Path dirPath = Paths.get( "data");
+    private static final Path filePath = Paths.get( "data", "data.txt");
+    private static final boolean directoryExists = Files.exists(dirPath);
+    private static final int LEN_LINE_SEPARATOR = 60;
+    public static final String LINE_SEPARATOR = "\t" + "-".repeat(LEN_LINE_SEPARATOR);
+
     public static void main(String[] args) {
```

Comment:
Consider keeping the main method as clean as possible. Shift the details of implementation into separate functions